### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by murphyke** [uwrit/leaf#606] [General]
@ndobb Is there a maintenance or other release planned? There are 14 unmerged dependabot PRs, and it has been over a year since the last release.

**Comment by ndobb** [comment]
@murphyke I'd encourage you to look at other branches, rather than simply the releases page. Also, I have begun a new job as an assistant professor, which greatly increased the work on my plate, so it's inevitable that rate of releases decreases (unless the community who use Leaf contribute back anything). None of that is to say development stops - Leaf is widely used, including at my previous institution and my current one.

**Comment by lrasmus** [comment]
@ndobb - for the dependabot PRs, would it be helpful to have someone test each that there are no breaking changes, and comment in the PR (tagging you?) if they look good to merge?  Or do you prefer to review package changes yourself?

**Reply by ndobb** [reply]
@lrasmus that would be amazing! I am still working on [Leaf development and planning a maintenance release](https://github.com/uwrit/leaf/tree/viz-fix), but certainly it would be a huge help if someone could check out the PRs. I don't necessarily prefer to review myself unless the changes are signficant
